common/util/Assert.cpp

```diff
@@ -3,7 +3,18 @@
 
 #include "Assert.h"
 
-void private_assert_failed(const char* expr, const char* file, int line, const char* function) {
-  fprintf(stderr, "%s:%d: Assertion failed: %s\nFunction: %s\n", file, line, expr, function);
+void private_assert_failed(const char* expr,
+                           const char* file,
+                           int line,
+                           const char* function,
+                           const char* msg) {
+  if ((msg != NULL) && (msg[0] == '\0')) {
```

Comment:
I think this condition should be `if( !msg || msg[0] == '\0')` so we don't try to print a message if its null.